app/models/concerns/has_many_common.rb

```diff
@@ -69,12 +70,12 @@ def has_many_names_for(association, options)
     end
 
     #### belongs_to ####
-    def belongs_to(association, options = {})
```

Comment:
http://apidock.com/rails/ActiveRecord/Associations/ClassMethods/belongs_to now accepts `scopes`, instead of `:conditions=>` , same thing with `has_many`